BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -300,12 +301,7 @@ public bool BacklightCompensation
             get
             {
                 CheckOptionSupported(RealSense2API.Option.BACKLIGHT_COMPENSATION, BacklightCompensationDesc.Name, RealSense2API.SensorName.COLOR);
-                float res = RealSense2API.GetOption(_pipeline, RealSense2API.SensorName.COLOR, RealSense2API.Option.BACKLIGHT_COMPENSATION);
-
-                if (res == 1.0f)
-                    return true;
-
-                return false;
+                return RealSense2API.GetOption(_pipeline, RealSense2API.SensorName.COLOR, RealSense2API.Option.BACKLIGHT_COMPENSATION) == 1.0f ? true : false;
```

Comment:
The `? true : false` is not needed.

If you want to write it in one line I would reverse the order of the comparison to make clear what the return value is compared against. As it is the line is too long to be displayed in Github CR.